components/webfield/ProgramChairConsole/Overview.js

```diff
@@ -458,8 +481,8 @@ const MetaReviewStatsRow = ({ pcConsoleData }) => {
           }
         />
         <StatContainer
-          title="AC Meta-Review Progress"
-          hint="% of area chairs who have completed meta reviews for all their assigned papers"
+          title={`${prettyField(singularAreaChairName)} Meta-Review Progress`}
```

Comment:
do we have a property for the meta review name?

Reply:
This is fixed